bin/addEntrypoint.bash

```diff
@@ -65,17 +74,13 @@ if [[ ! "$FEATURE_BRANCH_EXISTS" == "" ]]; then
   exit 255
 fi
 
-# Check if the directory for the entry point exits...
+# Check if the directory for the entry point exists...
 if [ -d "src/entrypoints/$ENTRY_POINT" ]; then
   echo -e "${ON_RED}ERROR:${NO_COLOR} A directory for the entry point $ENTRY_POINT already exists"
-  echo "in the directory $$ENTRY_POINT_SRC_DIR."
+  echo "in the directory $ENTRY_POINT_SRC_DIR."
   echo "Pick a different name for your entry point."
   echo "OR:"
   echo "  Remove the src/entrypoints/$ENTRY_POINT directory"
-  echo "  And remove any definitions related to the entry point $ENTRY_POINT from the files:"
```

Comment:
I think these lines need to remain as a reminder that simply removing the directory is not sufficient.